<launch>
    <rosparam command="load" file="$(find project11_vorc)/config/vorc.yaml" />
    
    <arg name="viz" default="true" />
    <arg name="camp" default="true" />

    <include file="$(find project11_vorc)/launch/localization.launch"/>
    
    
    <node pkg="cora_helm" type="cora_helm_node.py" name="cora_helm"/>

    <node pkg="cora_helm" type="simple_differential_controller.py" name="simple_controller">
        <remap from="odom" to="/cora/robot_localization/odometry/filtered"/>
    </node>

    <include if="$(arg viz)" file="$(find project11_vorc)/launch/viz.launch"/>

    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="autorepeat_rate" value="10.0"/>
    </node>
    
    <node pkg="cora_helm" type="joy_to_cmd_vel.py" name="joy_to_cmd"/>

    <node if="$(arg camp)" pkg="camp" type="AutonomousMissionPlanner" name="camp"  args="$(find project11_vorc)/charts/ $(find project11_vorc)/charts/L16-23800-34552-16-16_16.kap"/>
</launch>
